.github/workflows/push-test.yml

```diff
@@ -30,6 +30,11 @@ jobs:
           submodules: true
       - name: DiskANN Build CLI Applications
         uses: ./.github/actions/build
+  build-documentation:
+      strategy:
+        fail-fast: true
+      name: DiskANN Build Documentation
+      uses: ./.github/workflows/build-python-pdoc.yml
```

Comment:
Run the documentation generation on every build so we can download the docs and review them if needed.